lib/util/interpolator.ts

```diff
@@ -0,0 +1,132 @@
+import is from '@sindresorhus/is';
```

Comment:
this should have some tests here, so utils has full coverage if only run on that folder